### DISCUSSION TITLE
scrape http target with json?

### DISCUSSION THREAD

**Discussion by alpiua** [vectordotdev/vector#7708] [Q&A] [answered]
Didn't find a way to scrape a remote target over http with json support.
Does it exist ?

**Comment by spencergilbert** [comment, marked as answer]
Not a as a `source` today, I've seen a few users on our discord using the lua transform to make http calls - though I'm not sure that would fit your usecase

**Reply by jszwedko** [reply]
We will also be releasing an `exec` source shortly which will allow arbitrary execution of programs where vector will read their stdout/stderr and produce events. That program could make HTTP requests.

**Reply by alpiua** [reply]
> Not a as a `source` today, I've seen a few users on our discord using the lua transform to make http calls - though I'm not sure that would fit your usecase

Is it possible to see an example? Didn't check lua before.

**Reply by jszwedko** [reply]
I don't think that would work for your case @alpi-ua as we only support `lua` as a transform right now and you'd like to use it as a datasource.
